Makefile

```diff
@@ -10,7 +10,7 @@ ROOT=.
 FWDIR:=$(ROOT)/firmware
 BINDIR=$(ROOT)/bin
 SRCDIR=$(ROOT)/src
-INCDIR=$(ROOT)/include $(ROOT)/pros_include
```

Comment:
Mixing the headers and source together makes it clearer how .h and .cpp files mix together, which is something I've seen newer members struggle with firsthand. When you have parallel directories in different folders, it can be really hard to know that there's a separate declaration file, not every .h has a .cpp, etc.